### PR TITLE
Payment logos

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?

In order to bring a better experience to our users, we show the supported payment logos on the contributions page.

## Changes

* Added SVG Logos for the payment methods.
* Added an extra argument to contribution attributes to indicate if we should show the payment logos or not. 

## Screenshots
payment-logos

### CODE REVIEW

File: assets/components/inlinePaymentLogos/inlinePaymentLogos.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+// ----- Component ----- //
+
+export default function InlinePaymentLogos() {
+
+  return (
+    <div className={'component-inline-payment-logos'}>
+      <Svg svgName="visa-logo" />
+      <Svg svgName="mastercard-logo" />
+      <Svg svgName="paypal-logo" />
```
Comment (davidfurey) [inline assets/components/inlinePaymentLogos/inlinePaymentLogos.jsx]: paypal shouldn't be displayed if paypal is disabled

Comment (svillafe) [inline assets/components/inlinePaymentLogos/inlinePaymentLogos.jsx]: Good catch @davidfurey so far we want to turn on PayPal in all the channels, after this afternoon test with apps. We can see if we decide to turn off PayPal for that scenario. In that case, a following PR will be created.

**Comment by Amohkhan** [comment]
@svillafe 'VISA' text colour is incorrect, Paypal logo is not aligned with the others

**Comment by Amohkhan** [comment]
👍

**Pull request merged**